.vitepress/constants/constants.js

```diff
@@ -4,9 +4,10 @@ const constants = Object.freeze({
   nodeVersion: "21.7.2",
   yarnVersion: "1.22.19",
 
-  rollkitLatestTag: "v0.13.3",
+  rollkitLatestTag: "v0.14.3",
```

Comment:
should be v0.13.4: https://github.com/rollkit/rollkit/releases/tag/v0.13.4

Reply:
yeah, realized that